orchestra/communication/views.py

```diff
@@ -49,7 +55,36 @@ def reject_staffing_request_inquiry(request,
 @login_required
 def available_staffing_requests(request):
     worker = Worker.objects.get(user=request.user)
-    return render(request, 'communication/available_staffing_requests.html',
-                  {
-                      'requests': get_available_requests(worker),
-                  })
+    now = timezone.now()
```

Comment:
a) add tests for this
b) consider refactoring to make this more DRY
c) for discussion: is this modelform thing even worth it?